.eslintrc.js

```diff
@@ -13,7 +13,9 @@ module.exports = {
     sourceType: 'module'
   },
   plugins: ['react'],
-  rules: {},
+  rules: {
```

Comment:
I was getting the ESLint warnings in my environment
![image](https://user-images.githubusercontent.com/19779874/120927830-327d8d00-c71d-11eb-9b8f-9c25c7d03e10.png)